src/java/us/kbase/mobu/initializer/ModuleInitializer.java

```diff
@@ -18,13 +20,25 @@
 
 public class ModuleInitializer {
 	public static final String DEFAULT_LANGUAGE = "python";
+	private static final String KB_DEP = "KB_SDK_CONFIG";
 
 	private String moduleName;
 	private String userName;
 	private String language;
-	private boolean verbose;
 	private File dir;
-	
+	private static final String CFG_LOC ="kb-sdk";
```

Comment:
Generally good practice to group the constants and and class variables separately rather than intermixing them